app/assets/stylesheets/application.scss

```diff
@@ -72,6 +77,21 @@ a {
   }
 }
 
+.flash {
+  text-align: center;
+  padding: 10px;
```

Comment:
Suggestion only. All other measurements are in `rem` or `em`, which is helpful as it scales proportionately with the font size and makes it easier to keep a consistent look across different resolutions. What do you think about switching this to `1rem`?

Reply:
Done.